pydoclint/utils/walk.py

```diff
@@ -33,9 +33,9 @@ def walk(node):
 
 
 def walk_dfs(node):
-    """Depth-first traversal of AST. Modified from walk.walk, above."""
+    """Depth-first traversal of AST. Modified from `walk()` in this file"""
```

Comment:
Note: I changed the wording "above" because there's no guarantee that the relative positions of these 2 functions won't be changed in the future.